engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/bulk-actions-environment.html

```diff
@@ -41,7 +41,7 @@ <h4 translate>Assign Environment and Content View</h4>
 
       <button class="btn btn-default fr"
               translate
-              ng-hide="denied('edit_content_hosts', contentHost)"
+              ng-hide="denied('edit_hosts', contentHost)"
```

Comment:
For bulk actions there isn't 1 host, there is potentially thousands.  If you don't pass in an object it just checks to see that the user has that permission on something:

https://github.com/Katello/bastion/blob/master/app/assets/javascripts/bastion/auth/authorization.service.js#L22-L25

Which is better than nothing.  Ideally we might would ask the server out of the bulk selection and this permission how many hosts does the user have that permission on and display that, but that is for the future :)